### ISSUE TITLE
Android not building after react-native link

### ISSUE THREAD

**Issue by luchoster** [Kerumen/react-native-awesome-card-io#53]
Haven't even been able to run the example. I'm getting this error before running `react-native link`:
![screenshot_2017-08-06-21-59-49](https://user-images.githubusercontent.com/694367/29012843-59a21a10-7af3-11e7-94c8-c08a3b9b36fe.jpg)

after `react-native link` (can't even build it):
![screen shot 2017-08-06 at 22 05 57](https://user-images.githubusercontent.com/694367/29012861-77107682-7af3-11e7-85f6-f545a1989a26.png)

```
class CardIOExample extends Component {
  scanCard = () => {
    CardIOModule
      .scanCard({
      })
      .then(card => {
        console.log('card : ', card);
      })
      .catch(err => logger.error(`Failed to scan digital card. Error message: ${err}`));
  }

  render() {
    return (
      <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
        <TouchableOpacity onPress={() => this.scanCard()}>
          <Text>Scan card!</Text>
        </TouchableOpacity>
      </View>
    );
  }
}
```
my package.json:
"react": "16.0.0-alpha.12",
"react-native": "0.47.1",
"react-native-awesome-card-io": "^0.6.4"

I appreciate any help

**Comment by Kerumen** [comment]
Try to run `./gradlew clean` from the `android` folder.

**Comment by luchoster** [comment]
I did an didn't help at all :(

**Comment by luchoster** [comment]
actually if i comment 
```
@Override
    public List<Class<? extends JavaScriptModule>> createJSModules() {
      return Collections.emptyList();
    }
```
from `node_modules/react-native-awesome-card-io/android/src/main/java/com/cardio/RNCardIOPackage.java:23` the app runs and it works just fine. I am able to scan and get the card numbers.

**Comment by xavimb** [comment]
I'm having the same issue. It happened after upgrading to RN 0.47.0

**Comment by Kerumen** [comment]
Fixed by #54
